src/main/java/com/rakovets/course/java/core/practice/oop/principles/task02/heroes/Archer.java

```diff
@@ -0,0 +1,32 @@
+package com.rakovets.course.java.core.practice.oop.principles.task02.heroes;
+
+import com.rakovets.course.java.core.practice.oop.principles.task02.enemies.Enemy;
+import com.rakovets.course.java.core.practice.oop.principles.task02.heroSkills.AtackSkills;
+
+public class Archer extends Hero {
+
+    public Archer(String name, int health, int attack, AtackSkills atackSkills){
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"